src/components/Frame/components/ContextualSaveBar/tests/ContextualSaveBar.test.tsx

```diff
@@ -119,65 +116,72 @@ describe('<ContextualSaveBar />', () => {
     it('will render an image with the contextual save bar source', () => {
-      const contextualSaveBar = mountWithContext(<ContextualSaveBar />, {
-        logo: {
-          width: 200,
-          contextualSaveBarSource: './assets/monochrome_shopify.svg',
-        },
-      });
+      const contextualSaveBar = mountWithAppProvider(
+        <ContextualSaveBar />,
+        mergeThemeProviderContext({
+          logo: {
+            width: 200,
+            contextualSaveBarSource: './assets/monochrome_shopify.svg',
+          },
+        }),
+      );
       expect(contextualSaveBar.find(Image).prop('source')).toBe(
         './assets/monochrome_shopify.svg',
       );
     });
 
     it('will render an image with the width provided', () => {
-      const contextualSaveBar = mountWithContext(<ContextualSaveBar />, {
-        logo: {
-          width: 200,
-          contextualSaveBarSource: './assets/monochrome_shopify.svg',
-        },
-      });
+      const contextualSaveBar = mountWithAppProvider(
+        <ContextualSaveBar />,
+        mergeThemeProviderContext({
+          logo: {
+            width: 200,
+            contextualSaveBarSource: './assets/monochrome_shopify.svg',
+          },
+        }),
+      );
       expect(contextualSaveBar.find(Image).get(0).props.style).toHaveProperty(
         'width',
         '200px',
       );
     });
 
     it('will render the image with a default width if 0 is provided', () => {
-      const contextualSaveBar = mountWithContext(<ContextualSaveBar />, {
-        logo: {
-          contextualSaveBarSource: './assets/monochrome_shopify.svg',
-          width: 0,
-        },
-      });
+      const contextualSaveBar = mountWithAppProvider(
+        <ContextualSaveBar />,
+        mergeThemeProviderContext({
+          logo: {
+            contextualSaveBarSource: './assets/monochrome_shopify.svg',
+            width: 0,
+          },
+        }),
+      );
       expect(contextualSaveBar.find(Image).get(0).props.style).toHaveProperty(
         'width',
         '104px',
       );
     });
 
     it('will not render the logo when content is aligned flush left', () => {
-      const contextualSaveBar = mountWithContext(
+      const contextualSaveBar = mountWithAppProvider(
         <ContextualSaveBar alignContentFlush />,
-        {
+        mergeThemeProviderContext({
           logo: {
             contextualSaveBarSource: './assets/monochrome_shopify.svg',
             width: 200,
           },
-        },
+        }),
       );
 
       expect(contextualSaveBar.find(Image).exists()).toBeFalsy();
     });
   });
 });
 
-function mountWithContext(
-  node: React.ReactElement<any>,
-  polarisTheme?: ThemeProviderContext,
-) {
-  const context = polarisTheme ? polarisTheme : createThemeContext();
-  return mountWithAppProvider(
-    <ThemeProvider value={context}>{node}</ThemeProvider>,
-  );
+function mergeThemeProviderContext(providedThemeContext: ThemeProviderContext) {
```

Comment:
This might be overkill, it just saves writing `{context: themeProvider: {}}}` every time